.github/workflows/debug_aarch64.yml

```diff
@@ -64,6 +64,6 @@ jobs:
         uses: actions/upload-artifact@v2
         if: failure()
         with:
-          name: odroid_debug_arm64
+          name: failure-logs
```

Comment:
1. It does not look related to the commit 'ci: fix hardcoded artifact paths in workflows'.
2. AFAIU we can't set the same name for artifacts from different workflows, because they will rewrite each other. That's why they was named similarly to a workflow name.

(The same for the artifact name in `coverage.yml`.)

Reply:
It looks like the second item is relevant to workflows with a few jobs inside that upload artifacts [1], but not related to jobs running in different workflows. Uploading to the same artifact multiple times in the same workflow can overwrite the artifact.

[1] https://github.com/actions/upload-artifact/tree/v2#uploading-to-the-same-artifact

Reply:
TBH, I don't know for sure. I would appreciate if you'll monitor how it behave during some time.

Reply:
Ok, will do.